selftests/functional/test_unittest_compat.py

```diff
@@ -110,6 +110,9 @@ def test_run_error(self):
         self.assertIn('FAILED (errors=1)', result.stderr)
 
     def tearDown(self):
+        self.unittest_script_error.remove()
+        self.unittest_script_fail.remove()
+        self.unittest_script_good.remove()
```

Comment:
Not that I want to, I have to remove them, otherwise they are not removed until the `gc` runs the `__del__` which could be (and frequently is) after the next test is started...